### DISCUSSION TITLE
inpaint mask color

### DISCUSSION THREAD

**Discussion by reese555** [lllyasviel/Fooocus#1346] [Ideas]
probably dumb question but any easy way to change mask color from white?

**Comment by xhoxye** [comment]
Why change it?

**Reply by reese555** [reply]
a preference/eyesight thing. its not a huge deal but some of us with more sensitive eyes with intense colors develop headaches etc especially if using it extended periods of time. like i said not a big deal would just be a nice to have for some of us.
